README.md

```diff
@@ -37,3 +37,9 @@ TODO
 
 ## SWITCH A C++
 TODO
+
+## Weather_App
+1. Create a .env file inside weather_app/
```

Comment:
Consider adding a .env.example to leave a schema of what the .env should look like.